hwdb.d/70-hw-wallets.hwdb

```diff
@@ -0,0 +1,24 @@
+# This file is part of systemd.
+#
+# Database for Hardware Wallets that should be accessible to the seat owner.
+##
+# To add local entries, copy this file to
+#   /etc/udev/hwdb.d/
+# and add your rules there. To load the new rules execute (as root):
+#   systemd-hwdb update
+#   udevadm trigger
+
+################
+# Trezor HW Wallets
+################
+
+# Trezor v1
+usb:v534Cp0001*
+ ID_TREZOR_HW=1
+
+# Trezor v2
+usb:v1209p53C0*
+ ID_TREZOR_HW=1
+
+usb:v1209p53C1*
+ ID_TREZOR_HW=1
```

Comment:
This isn't a "category"? Just call it "HW_WALLET"?

Reply:
Maybe `HARDWARE_WALLET` instead of `HW_WALLET`?

Also the file could be named `70-hardware-wallets.hwdb` instead?

@YHNdnzj thougts?

Reply:
Added another fixup commit where it is changed to `ID_HARDWARE_WALLET` and hwdb file to `70-hardware-wallets.hwdb`